project/cmake/scripts/windows/Macros.cmake

```diff
@@ -14,9 +15,15 @@ function(core_link_library lib wraplib)
     endforeach()
 
     add_library(wrap_${lib} SHARED ${SOURCES_ABS})
-    set_target_properties(wrap_${lib} PROPERTIES OUTPUT_NAME ${wraplib})
+    set_target_properties(wrap_${lib} PROPERTIES OUTPUT_NAME lib${lib})
     target_include_directories(wrap_${lib} PRIVATE ${INCLUDES})
     target_compile_definitions(wrap_${lib} PRIVATE ${DEFINITIONS})
+    
+    if(MSVC)
```

Comment:
is that needed? since this file is used on windows only. I'm not sure if MSVC is set only when using the Visual Studio generator, but I'll try to test that in the next days.